src/storm/templates/configmap.yaml

```diff
@@ -25,6 +25,7 @@ data:
     supervisor.childopts: {{ $.Values.supervisor.childopts }}
     supervisor.worker.heap.memory.mb: {{ $.Values.supervisor.worker.heap_memory_mb }}
```

Comment:
I am under the impression that this has been the wrong option all along - it should have been `worker.heap.memory.mb` instead. See [here](https://github.com/apache/storm/blob/v1.2.3/conf/defaults.yaml#L173) for v1.2.3 and [here](https://github.com/apache/storm/blob/v2.2.0/conf/defaults.yaml#L193) for v2.2.0.

@yasram1 @mariannovotny Would you agree?

PS: this is _not_ a real comment about this PR — I am just taking advantage of it for checking this potential issue

Reply:
Hi @jgiannuzzi We had issue with worker default java.options that are not supported in java 11. The defaults are here 
https://github.com/apache/storm/blob/master/conf/defaults.yaml#L195

Reply:
sure - I understand that and have no issue with it! I am talking about a separate option here: `supervisor.worker.heap.memory.mb`, which I believe should instead be `worker.heap.memory.mb`. I wanted to discuss it here before creating any issue/PR about it.

Reply:
Yes it looks as not valid setting in the current config (1.2.3+ ) and it is ignored. I think "worker.heap.memory.mb" can be useful to have